package.json

```diff
@@ -14,7 +14,11 @@
     "npm": "1.3"
   },
   "dependencies": {
+<<<<<<< HEAD
```

Comment:
You commited a conflict, could you please clean this ? (and I don't think we need to update docpad).